README.md

```diff
@@ -1,2 +1,73 @@
 # jboss-eap-8-openshift-image
  OpenShift container images for Red Hat JBoss Enterprise Application Platform 8
+
+This project defines Images allowing you to build and deploy EAP 8 applications on the cloud.
+
+EAP 8 S2I (Source-To-Image) builder images:
```

Comment:
We should write somewhere that the builder and runtime images are not meant to be mixed (or more positively that builder and runtime images must share the same JDK version)